_translator-files/po/de/FAQ.po

```diff
@@ -45,159 +47,161 @@ msgid ""
-msgstr ""
+msgstr "Nein. Die Musiker auf einem Jamulus-Server spielen einfach in Echtzeit zusammen, so wie sie es auch persönlich tun würden. Wenn du ein Zeitsignal wünscht, kannst du im Internet Lösungen für gemeinsame Metronome ausprobieren. Aber es ist wahrscheinlich am besten, die Latenz zu minimieren, so dass niemand eine Gesamtverzögerung von mehr als 30-50 ms hat."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:28
 msgid "Bear in mind also that all participants should follow [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) which will also determine whether you can play in time properly."
-msgstr ""
+msgstr "Denke auch daran, dass sich alle Teilnehmer an die [Goldene Regel](/wiki/Client-Troubleshooting#der-sound-ist-gut-aber-es-ist-schwierig-im-takt-zu-bleiben) halten sollten, denn davon hängt auch ab, ob alle richtig im Takt spielen können."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:30
 msgid "### How do I know if I can join a Server? Are there rules?"
-msgstr ""
+msgstr "### Woher weiß ich, ob ich einem Server beitreten kann? Gibt es Regeln?"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:32
 msgid "In general, if somebody lists a Server on one of the Public Directories provided by Jamulus by default, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms built in. However, some Servers may state their policies in the welcome message you will see in the chat window."
-msgstr ""
+msgstr "Wenn jemand einen Server in eines der öffentlichen Verzeichnisse einträgt, die Jamulus standardmäßig bereitstellt, akzeptiert er damit, dass jeder auf ihm spielen kann. Jamulus hat keinen Passwortschutz oder andere Authentifizierungsmechanismen eingebaut. Einige Server können jedoch in der Willkommensnachricht, die du im Chat-Fenster sehen kannst, Richtlinien angeben."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:34
 msgid "Note also that Servers do not have to be registered on a Directory in order for Jamulus Clients to connect to them. Server operators can simply give out the address of their Servers to those they want to play with, and they can then connect directly. Have a look at [this overview for more information](/wiki/Running-a-Server#server-types)."
-msgstr ""
+msgstr "Beachte auch, dass die Server nicht in einem Verzeichnis registriert sein müssen, damit Jamulus-Clients sich mit ihnen verbinden können. Serverbetreiber können einfach die Adresse deines Servers an diejenigen weitergeben, mit denen sie spielen wollen, und diese können sich dann direkt verbinden. Schaue dir [diese Übersicht für weitere Informationen](/wiki/Running-a-Server#server-typen) an."
```

Comment:
Make clear what "ihnen verbinden können" refers to. 
In short maybe something like 

Beachte, dass ein Server nicht an einem Verzeichnis registriert sein muss, damit Jamulus Clients sich (mit dem  Server) verbinden können.

Reply:
Corrected